src/lib/rng/jitter_rng/jitter_rng.h

```diff
@@ -0,0 +1,44 @@
+/*
+* CPU Jitter Random Number Generator
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_JITTER_RNG_H_
+#define BOTAN_JITTER_RNG_H_
+
+#include <botan/rng.h>
+
+namespace Botan {
+
+namespace {
+
+struct Jitter_RNG_Internal;
+
+}
+
+/*
+* RNG using libjitterentropy (https://github.com/smuellerDD/jitterentropy-library).
+*/
+class BOTAN_PUBLIC_API(3, 6) Jitter_RNG final : public RandomNumberGenerator {
+   public:
+      Jitter_RNG();
+      ~Jitter_RNG();
+
+      std::string name() const override { return "JitterRNG"; }
+
+      bool is_seeded() const override { return true; }
+
+      bool accepts_input() const override { return false; }
+
+      void clear() override {}
```

Comment:
Perhaps it would make sense to re-set the `m_jitter` internal implementation on `clear()`?

Essentially (in the cpp file): `m_jitter = std::make_unique<Jitter_RNG_Internal>()`. Note that the `JitterRNG()` constructor could then simply call `clear()` internally without previously allocating `m_jitter` in its initializer list.